config/settings.py

```diff
@@ -3,7 +3,7 @@
 
 class Settings(BaseSettings):
     database_url: str = "postgresql+asyncpg://authdb_owner:V97gEkaPchLf@ep-plain-water-a5scbycn.us-east-2.aws.neon.tech/authdb"
-    secret_key: str = "e705c2325058744e7328d842be8da79a1f7c934363b1f061c7934d4804f3d360"
+    secret_key: str = "empty_string_1"
     algorithm: str = "HS256"
```

Comment:
Hardcoding sensitive information such as database URLs and secret keys in the source code is a security risk. This can lead to accidental exposure of credentials and secret keys if the codebase is shared or becomes public. It's recommended to use environment variables for such sensitive information and load them at runtime.

**Label**: security

**Code suggestion**

```python
from os import getenv

class Settings(BaseSettings):
    database_url: str = getenv('DATABASE_URL', 'default_database_url_here')
    secret_key: str = getenv('SECRET_KEY', 'default_secret_key_here')
```